cmobase/introduction.adoc

```diff
@@ -2,8 +2,10 @@
 == Introduction
 
 _Cache management operation_ (or _CMO_) instructions perform operations on
-cached copies of data in the memory hierarchy. These instructions are grouped by
-operation into the following instruction classes:
+copies of data in the memory hierarchy. In general, CMO instructions operate on
+cached copies of data, but in some cases, a CMO instruction may operate on the
+copy of data in memory. Furthermore, CMO instructions are grouped by operation
```

Comment:
In the context of the MMIO use case for CBO.ZERO, would recommend reducing ambiguity here introduced by the phrase "copy of data in memory" by relying on the previous "memory location" definition in the specification, restating this sentence along the lines of:

In general, CMO instructions operate on cached copies of data; but in some cases, a CMO instruction may operate directly on the physical resources identified by memory locations.

Reply:
Fair. Will update.